rapids-cmake/cuda/init_runtime.cmake

```diff
@@ -43,9 +43,13 @@ function(rapids_cuda_init_runtime use_static value)
 
   if(NOT DEFINED CMAKE_CUDA_RUNTIME_LIBRARY)
     if(${value})
-      set(CMAKE_CUDA_RUNTIME_LIBRARY "Static" PARENT_SCOPE)
+      # Control legacy FindCUDA.cmake behavior too
+      set(CUDA_USE_STATIC_CUDA_RUNTIME ON PARENT_SCOPE)
```

Comment:
Please update the docs to state the function will also set `CUDA_USE_STATIC_CUDA_RUNTIME`

Reply:
Done